src/ktn_dodger.erl

```diff
@@ -715,10 +718,10 @@ scan_form([{'?', L}, {Type, _, _} = N | [{'(', _} | _] = Ts], Opt)
 scan_form(Ts, Opt) ->
     scan_macros(Ts, Opt).
 
-build_info_string(Prefix, Ts0) ->
+build_info_string(PreFix, Ts0) ->
```

Comment:
It's not a _previous fix_ (i.e., PreFix)… it's a _prefix_ … single noun ;)

Reply:
I understand, but it was `elvis` that said it 😄 so I "fixed" it. I gotta come up with an exception 😄 Or I could otherwise move the other `PreFix` and `PostFix` to `Prefix` and `Postfix`. 😄

Reply:
`consistent_variable_casing` doesn't seem to support options 😢

Reply:
yeah… sadly… you'll have to ignore this file altogether for that rule.

Reply:
9e21408